prober/handler.go

```diff
@@ -176,6 +179,15 @@ func (sl scrapeLogger) Log(keyvals ...interface{}) error {
 	sl.bufferLogger.Log(keyvals...)
 	kvs := make([]interface{}, len(keyvals))
 	copy(kvs, keyvals)
+
+	if sl.logErrors {
+		for i := 0; i < len(kvs); i += 2 {
+			if kvs[i] == level.Key() && kvs[i+1] == level.ErrorValue() {
```

Comment:
this will only log errors from the checks? maybe we can make this configurable where users can configure the level they want to log? 

this would make this flag bit generic and more useful. we can rename the flag to something like `--log.prober=info`, and you can configure it to `--log.prober=error` to log only error, and replicate what we have in current state.

IMO this feature will be useful to users who are running BBE with central log collection in place.

we can set it to `error` by default or disable it by default, I am fine with both options.

cc @SuperQ what do you think?

Reply:
What if `--log.prober=info`, but  `--log.level=warn` if set? 

Should info logs still visible on stdout?

Reply:
@electron0zero @SuperQ how we can proceed here?